tests/integration/cmd_dev_test.go

```diff
@@ -303,6 +304,12 @@ var _ = Describe("odo dev command tests", func() {
 						ContainSubstring(fmt.Sprintf("%s-app-", cmpName)),
 					))
 				})
+
+				// Returned pvc yaml contains ownerreference
+				By("creating a pvc with ownerreference", func() {
+					output := commonVar.CliRunner.Run("get", "pvc", "--namespace", commonVar.Project, "-o", `jsonpath='{range .items[*].metadata.ownerReferences[*]}{@..kind}{"/"}{@..name}{end}'`).Out.Contents()
```

Comment:
I just tested this command, and without a break line after each element, if there are multiple PVCs, we might everything printed on a single line and with no separator.

```suggestion
					output := commonVar.CliRunner.Run("get", "pvc", "--namespace", commonVar.Project, "-o", `jsonpath='{range .items[*].metadata.ownerReferences[*]}{@..kind}{"/"}{@..name}{"\n"}{end}'`).Out.Contents()
```